handlers/spaces.py

```
@@ -68,9 +68,9 @@ async def check_transaction_status(signature: str, command_start_time: datetime,
         # Convert signature string to Signature object
         try:
             signature_obj = Signature.from_string(signature)
-        except Exception as e:
+        except Exception as original_exception:
             logger.error(f"Error converting signature format: {str(e)}")
```

Ruff (0.8.2)

Undefined name `e`

(F821)